edge-apps/google-calendar/README.md

```diff
@@ -0,0 +1,191 @@
+# Screenly Google Calendar App
+
+## Prerequisites
+
+- Node.js (v22.0.0+)
+- npm (v10.9.0+)
+- Screenly Edge App CLI (v1.0.3+)
+
+## Getting Started
+
+```bash
+$ cd edge-apps/google-calendar
+$ screenly edge-app create \
+    --name=EDGE_APP_NAME \
+    --in-place
+$ screenly edge-app instance create
```

Comment:
This isn't necessary.

Reply:
@vpetersson, I think the `instance create` command is still necessary at the moment if the calendar app needs to be installed. Unless an Edge App can be created via the web console, that line can be removed.